### ISSUE TITLE
Data provider

### ISSUE THREAD

**Issue by plumpudding** [plumpudding/hopglass-server#13]
Something to run on supernodes..
Problem: adding to a multicast group doesn't seem to work.

**Comment by anoymouserver** [comment]
ich glaube an sowas versucht sich @sunz3r gerade.

**Comment by Sunz3r** [comment]
Ja, bin daran aber fehlen noch ein paar Daten (z.B. neighbours), ansonsten ist statistics und nodeinfo fertig und wird ordentlich vom HopGlass-Server verarbeitet. Es gibt aber schon eine zweite Gruppe die unabhängig daran arbeitet:
https://github.com/FreifunkBremen/respondd

**Comment by Sunz3r** [comment]
Habe mit @anoymouserver zusammen den Data provider soweit fertig, sind sicher noch ein paar Bugs drin aber es läuft (getestet auf: Gateway, Offloader (mit WLAN- und Kabel-Mesh) und Server) und macht was er soll:
https://github.com/Sunz3r/ext-respondd

**Comment by plumpudding** [comment]
OK. Mal sehen, ob wir das in der Readme verlinken oder so. Ich würde es aber auf jeden Fall als seperates Projekt betreiben und in Zukunft noch einen Hopglass Responder anbieten. Jetzt da es ein funktionierendes Programm zu diesem Zweck gibt, rückt das allerdings ganz nach unten auf der TODO. Ich lasse dieses Issue offen.
